test/WebSites/ControllersFromServicesClassLibrary/ResourcesController.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace ControllersFromServicesClassLibrary
 {
+    [Controller]
```

Comment:
In the real world, should users add `[Controller]` on the base `ResourcesController` or on their `Inventory` controller? Realize both work but what would we recommend? Whichever that is, do it here.